packages/grafana-ui/src/components/Logs/LogRow.tsx

```diff
@@ -127,6 +128,13 @@ class UnThemedLogRow extends PureComponent<Props, State> {
     });
   };
 
+  renderTimeStamp(epochMs: number) {
+    return dateTimeFormat(epochMs, {
+      timeZone: this.props.timeZone,
+      format: defaultDateFormats.fullDate,
```

Comment:
We don't need to specify this here. It will use the `fullDate` as default - right? But maybe it is good to be explicit.